src/handlebar-helpers.js

```diff
@@ -29,6 +29,14 @@ const helpers = {
       return data.split(delimiter);
     }
     return [data];
+  },
+  divide: function (v1, v2) {
+    if ( v2 != 0 ) {
+      return parseInt(v1 / v2);
+    }
+    else{
+      return "can not divide as v2 is 0"
+    }
```

Comment:
i dont think its appropriate to return a string when the template is expecting a number. lets just return a default value of 0 or -1 to show an error has occurred. 
```suggestion
    } else {
      return -1;
    }
```